Dockerfile

```diff
@@ -9,4 +9,9 @@ RUN sed -i -e 's!\bmain\b!main contrib!g' /etc/apt/sources.list && \
     && clean-install \
     xfsprogs \
     open-iscsi \
-    azure-cli \
+    azure-cli
+
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list &&\
```

Comment:
Is this purposely done on a new line? Adding the repo and using `clean-install -t stretch-backports $package` should also work.

If it needs to be on a separate line, please keep the same format as above:

```
$VAR1 \
&& $VAR2
```

Or change all to the new style.